<!DOCTYPE HTML>
<html>

<head>
  <title>Hongyuan Cao's Homepage</title>
  <meta name="description" content="website description" />
  <meta name="keywords" content="website keywords, website keywords" />
  <meta http-equiv="content-type" content="text/html; charset=windows-1252" />
  <link rel="stylesheet" type="text/css" href="template/style.css" title="style" />
</head>

<body>
  <div id="main">
    <div id="header">
      <div id="logo">
        <div id="logo_text">
          <!-- class="logo_colour", allows you to change the colour of the text -->
          <h1>Hongyuan Cao</h1>
          <h2>Department of Statistics</h2>
          <h2>Florida State University</h2>
        </div>
      </div>
      <div id="menubar">
        <ul id="menu">
          <!-- put class="selected" in the li tag for the selected page - to highlight which page you're on -->
          <li><a href="index.html">Home</a></li>
          <li class="selected"><a href="Hongyuan_publication.html">Publication</a></li>
          <li><a href="Hongyuan_software.html">Software</a></li>
          <li><a href="Hongyuan_teaching.html">Teaching</a></li>
        </ul>
      </div>
    </div>
    <div id="site_content">
      <div id="sidebar_container">
        <div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <!-- insert your sidebar items here -->
			<img src="Hongyuan_Cao.jpg" width="195" align="middle">
          </div>
          <div class="sidebar_base"></div>
	<div class="sidebar">
          <div class="sidebar_top"></div>
          <div class="sidebar_item">
            <h3>Useful Links</h3>
            <ul>
              <li><a href="https://github.com/hongyuan-cao">My GitHub Page</a></li>
              <li><a href="https://scholar.google.com/citations?user=SIEOANEAAAAJ&hl=en">My Google Scholar Page</a></li>
            </ul>
          </div>
          <div class="sidebar_base"></div>
        </div>
        </div>
      </div>
      <div id="content">
		<h2>Methodological papers</h2>
		<ul>
		  
<li>Li, Y., Zhou, X., Chen, R., Zhang, X. and <strong>Cao, H. </strong> (2024): STAREG: statistical replicability analysis of high throughput experiments with applications to spatial transcriptomic studies.  <br /> 
             <I>PLOS Genetics</I>, to appear. <br />
	 <a href = "https://github.com/hongyuan-cao/STAREG-Analysis">Download the R codes</a>	  
<li>Li, Y., Lei, H., Wen, X., and <strong>Cao, H. </strong>  (2024): A powerful approach to identify replicable variants in genome-wide association studies. <br /> 
             <I>American Journal of Human Genetics</I>, 111, 966-978.<br />
	 <a href = "https://github.com/hongyuan-cao/ReAD">Download the R codes</a>	
<li>
<li>Sun, Z., <strong>Cao, H.</strong>, Cao, H., Chen, L. and Fine, J. P. (2024): Analysis of longitudinal data with omitted asynchronous longitudinal covariate.  <br /> 
            <I>Journal of Statistical Planning and Inference</I>, 231, 106135.<br />
<li>Liu, C., Sun, Z. and <strong>Cao, H.</strong> (2023): Regression analysis of mixed sparse synchronous and asynchronous longitudinal data in varying-coefficient regression models. <br /> 
            <I>Electronic Journal of Statistics</I>, 17, 3103-3142.<br />
	
<li>Lyu, P., Li, Y., Wen, X. and <strong>Cao, H.</strong> (2023): JUMP: replicability analysis of high-throughput experiments with applications to spatial transcriptomic studies.  <br /> 
            <I>Bioinformatics</I>, 39(6).<br />
	
<li>Boadu, F., <strong>Cao, H.</strong> and Cheng, J. (2023): Combining protein sequences and structures with transformers and equivariant graph neural networks to predict protein function.  <br /> 
            <I>Bioinformatics</I>, 39, i318-i325.<br />
	
<li> <strong>Cao, H.,</strong> Chen, J. and Zhang, X. (2022): Optimal false discovery rate control for large-scale multiple testing with auxiliary information. <br /> 
            <I>Annals of Statistics</I>, 50, 807-857.<br />

<li> Li, Y., Zhou, X. and <strong>Cao, H.,</strong> (2022): Statistical analysis of spatially resolved transcriptomic data by incorporating multi-omics auxiliary information. <br /> 
            <I>Genetics</I>, 221(4).<br />
	
<li> Sun, Z., <strong>Cao, H.,</strong> and Chen, L. (2022): Regression analysis of additive hazards model with sparse longitudinal covariates. <br /> 
            <I>Lifetime Data Analysis</I>, 28, 263-281. <br />
	
<li><strong>Cao, H.</strong> and Wu, W. (2022): Testing and estimation for clustered signals. <br />
			<I>Bernoulli</I>, 28, 525-547. <br />
            
<li><strong>Cao, H. </strong> and Fine, J. P. (2021): On the proportional hazards model with last observation carried forward covariates.<br />
			<I>Annals of the Institute of Statistical Mathematics</I>, 73, 115-134 <br />
           
<li>Li, H., Tian, G., Tang, N. and <strong>Cao, H.</strong> (2018): Assessing non-inferiority for incomplete paired data under non-ignorable missing mechanism.  <br />
			<I>Computational Statistics and Data Analysis </I>, 127, 69-71.

<li>Xiao, J.,  <strong>Cao, H.</strong> and Chen, J. (2017): False discovery rate control incorporating phylogenetic tree increases detection power in microbiome-wide multiple testing.  <br />
			<I>Bioinformatics</I>, 33, 2873-2881. <br />
	
<li><strong>Cao, H. </strong>, Sen, P., Peery, A. F. and Dellon, E. S. (2016): Assessing agreement with multiple raters on correlated kappa statistics.<br />
			<I>Biometrical Journal</I>, 58, 935-943 <br />
           
<li><strong>Cao, H.</strong>, Li, J. and Fine, J. P. (2016): On last observation carried forward and asynchronous longitudinal regression analysis.  <br />
			<I>Electronic Journal of Statistics</I>, 10, 1155-1180.

<li> <strong>Cao, H.</strong> and Wu, W. (2015): Changepoint estimation: another look at multiple testing problems.  <br />
			<I>Biometrika</I>, 102, 974-980. <br />
	
<li><strong>Cao, H. </strong>, Churpek, M. M., Zeng, D. and Fine, J. P. (2015): Analysis of the proportional hazards model with sparse longitudinal covariates.<br />
			<I>Journal of the American Statistical Association-Thoery and Method</I>, 110, 1187-1196 <br />
           
<li><strong>Cao, H.</strong>, Zeng, D. and Fine, J. P. (2015): Regression analysis of sparse asynchronous longitudinal data.  <br />
			<I>Journal of the Royal Statistical Society, Series B</I>, 77, 755-776.

<li> <strong>Cao, H.</strong> and Kosorok, M. (2015): The optimal power puzzle: scrutiny of the monotone likelihood ratio assumption in multiple testing.  <br />
			<I>Biometrika</I>, 100, 495-502. <br />
	
<li><strong>Cao, H. </strong>, LaVange, L. M., Heyse, J. F., Mast, T. C., Kosorok, M. R. (2013). Medical records-based postmarketing safety evaluation of rare events 
	with uncertain status.<br /> <I> Journal of Biopharmaceutical Statistics </I>, 23, 201-212.
	
<li> Lan, Y., Lin, T. and <strong> Cao, H. </strong> (2012). Evaluating two PTI test procedures for control of delivered dose uniformity for aerosol products. <br />
<I> Statistics in Biopharmaceutical Research</I>, 4, 136-148.


<li> <strong>Cao, H.</strong> and Kosorok, M. (2011): Simultaneous critical values for t-tests in very high dimensions. <br />
			<I>Bernoulli</I>, 17, 347-394. <br />
	
<li> <strong>Cao, H.</strong> (2007): Moderate deviations for two sample t-statistics.  <br />
			<I>European Series in Applied and Industrial Mathematics: Probability and Statistics</I>, 11, 264-271. <br />
	
           
			</li>
			
		</ul>
		<h2>Collaborative papers</h2>
		<ul>
			<li> <a href = "http://repository.upenn.edu/edissertations/2013/">Statistical methods for compositional and tree-structured count data in human microbiome studies</a><br />
			<I>PhD Thesis</I>, 2016.</li>
			<li> <strong>Shi, P.</strong> and Zhang, A.: <a href = "http://arxiv.org/abs/1305.4896">Methods to calculate the upper bound of Gini coefficient based on grouped data and the result for China</a><br />
			<I>preprint of Institute of Mathematics, Peking University</I>, 2010-20.</li>
		</ul>
      </div>
    </div>
    <div id="content_footer"></div>
    <div id="footer">
      Copyright &copy; textured_stars_light | <a href="http://validator.w3.org/check?uri=referer">HTML5</a> | <a href="http://jigsaw.w3.org/css-validator/check/referer">CSS</a> | <a href="http://www.html5webtemplates.co.uk">design from HTML5webtemplates.co.uk</a>
    </div>
  </div>
<!-- Start of StatCounter Code for Default Guide -->
<script type="text/javascript">
var sc_project=11414960;
var sc_invisible=1;
var sc_security="e117e79e";
var scJsHost = (("https:" == document.location.protocol) ?
"https://secure." : "http://www.");
document.write("<sc"+"ript type='text/javascript' src='" + scJsHost+
"statcounter.com/counter/counter.js'></"+"script>");
</script>
<noscript><div class="statcounter"><a title="web analytics"
href="http://statcounter.com/" target="_blank"><img class="statcounter"
src="//c.statcounter.com/11414960/0/e117e79e/1/" alt="web
analytics"></a></div></noscript>
<!-- End of StatCounter Code for Default Guide -->
</body>
</html>
